main_app/templates/main_app/women_laws.html

```diff
@@ -255,14 +255,22 @@ <h5>Dowry Prohibition Act(1961) </h5>
             <li>
                 <div class="collapsible-header">
                     <i class="material-icons">arrow_drop_down</i>
-                    <h5>Indian Divorce Act(1969) </h5>
+                    <h5>Indian Divorce Act(1869) </h5>
                 </div>
                 <div class="collapsible-body">
-                    <p>The Indian Divorce Act allows the dissolution of marriage, mutual consent, nullity of marriage,
-                        judicial
-                        separation and restitution of conjugal rights. Family Courts are established to file, hear, and
-                        dispose of such
-                        cases.</p>
+                    <p>The Divorce Act 1869 provides for the grounds for dissolution of marriage. Both the husband and wife can move to court to seek a decree for divorce under this act.  
```

Comment:
Please add bullets for the content that is big enough to read. So that it will be in an organized manner.